rhacmstackem.sh

```diff
@@ -129,6 +131,9 @@ sleep 600
 # Point to claimed cluster
 export KUBECONFIG=${LIFEGUARD_PATH}/clusterclaims/${CLUSTERCLAIM_NAME}/kubeconfig
 
+# Add certificate
+./cluster-cert/apply-apps-cert.sh "${CLUSTERCLAIM_NAME}" "${HOSTED_ZONE_NAME}" 
```

Comment:
I don't think CLUSTERCLAIM_NAME is correct here. I think you want the ClusterDeployment name.